README.md

```diff
@@ -1,2 +1,175 @@
-# bibernate
-Bibernate ORM framework
+<p align="center">
+    <img src="doc/bibernate.png" width="50%" alt="Bibernate"/>
+</p>
+
+*Bibernate* is a custom simplified Object-Relational Mapping framework. It helps to work with database in object-oriented manner.
+
+## Main features
+- Configuration
+  - Read data source configuration from yaml file
```

Comment:
We can also mention somewhere here that multiple persistence units are supported in one file, as well as multiple persistence files (this is also a feature :))